cc/pass_alive_test.cc

```diff
@@ -0,0 +1,732 @@
+          ..XXXXXX.
+          ..XXXXXX.
+          .........
+          .........
+          .........
+          .........
+          .........)"},
+
+      {// board state
+       R"(.........
+          .OOOOOOO.
+          .O.OXXXO.
+          .OOOXXXO.
+          .OX.XXXO.
+          .OXXXXOO.
+          .OOOOOO..
+          .........
+          .........)",
+       // expected result
+       R"(.........
+          .OOOOOOO.
+          .OOOOOOO.
+          .OOOOOOO.
+          .OOOOOOO.
+          .OOOOOOO.
+          .OOOOOO..
+          .........
+          .........)"},
+
+      {// board state
+       R"(.........
+          .........
+          .........
+          .........
+          .........
+          .........
+          OOOOO....
+          .O.XOO...
+          O.O.XO...)",
+       // expected result
+       R"(.........
+          .........
+          .........
+          .........
+          .........
+          .........
+          OOOOO....
+          OOOOOO...
+          OOOOOO...)"},
+
+      {// board state
+       R"(OXX.X.X.O
+          OX.XOXXXX
+          OXXOOOOOO
+          OOOO.....
+          .........
+          .........
+          .........
+          .........
+          .........)",
+       // expected result
+       R"(OXXXXXXXX
+          OXXXOXXXX
+          OXXOOOOOO
+          OOOO.....
+          .........
+          .........
+          .........
+          .........)"},
+
+      {// board state
+       R"(..O.O.XX.
+          .OOOO.X.X
+          OO.....XX
+          .........
+          .........
+          X......OO
+          .XX...O.O
+          X.X..OO.O
+          .X.X.O.O.)",
+       // expected result
+       R"(..O.O.XXX
+          .OOOO.XXX
+          OO.....XX
+          .........
+          .........
+          X......OO
+          .XX...OOO
+          X.X..OOOO
+          .X.X.OOOO)"},
+
+      {// board state
+       R"(.OOOOOOO.
+          OOX..XX.O
+          O...XOX.O
+          O...X.X.O
+          OXXX.XX.O
+          OXO.X...O
+          OX.XX.XXO
+          O.XX..OOO
+          .OOOOOOX.)",
+       // expected result
+       R"(OOOOOOOOO
+          OOX..XX.O
+          O...XXX.O
+          O...XXX.O
+          OXXXXXX.O
+          OXXXX...O
+          OXXXX.XXO
+          O.XX..OOO
+          OOOOOOOOO)"},
+
+      {// board state
+       R"(.........
+          .........
+          .........
+          .........
+          .........
+          .........
+          .........
+          .........
+          .........)",
+       // expected result
+       R"(.........
+          .........
+          .........
+          .........
+          .........
+          .........
+          .........
+          .........
+          .........)"},
+
+      {// board state
+       R"(.........
+          .........
+          ...XX....
+          ..X..X...
+          ..X..X...
+          ...XX....
+          .........
+          .........
+          .........)",
+       // expected result
```

Comment:
the entire board isn't just considered dead?

Reply:
"Not pass-alive" would be more accurate than "dead".

The original black stones are in the expected result because of how the test is set up: it overlays the result from `CalculatePassAliveRegions` onto the input board position. In this case, `CalculatePassAliveRegions` returns an empty result for both black & white, so the expected result is just the input result.

I think we may be saying the same thing...

Reply:
"Not pass-alive" would be more accurate than "dead".

The original black stones are in the expected result because of how the test is set up: it overlays the result from `CalculatePassAliveRegions` onto the input board position. In this case, `CalculatePassAliveRegions` returns an empty result for both black & white, so the expected result is just the input result.

I think we may be saying the same thing...